### PR TITLE
webui: update dependencies

### PR DESCRIPTION
**Pull request by or2e**

### CODE REVIEW

File: src/WebUI/package.json [hunk 1]
```diff
@@ -29,61 +28,57 @@
...
+    "qs": "^6.11.2",
+    "ufo": "^1.1.2",
+    "vue": "^3.3.2",
     "vue-i18n": "^9.2.2",
-    "vue-router": "^4.1.6",
+    "vue-router": "^4.2.0",
     "vue-slider-component": "^4.1.0-beta.7"
   },
   "devDependencies": {
-    "@intlify/vite-plugin-vue-i18n": "^6.0.3",
```
Comment (or2e) [inline src/WebUI/package.json:42]: `@intlify/vite-plugin-vue-i18n` - deprecated

File: src/WebUI/package.json [hunk 2]
```diff
@@ -29,61 +28,57 @@
...
-    "vite": "^4.3.4",
+    "vite": "^4.3.7",
     "vite-plugin-compression": "^0.5.1",
     "vite-plugin-svg-icons": "^2.0.1",
     "vite-plugin-top-level-await": "^1.3.0",
     "vite-plugin-vue-layouts": "^0.8.0",
-    "vitest": "^0.31.0",
-    "vue-tsc": "^1.6.3"
-  },
-  "overrides": {
```
Comment (or2e) [inline src/WebUI/package.json:85]: no longer needed

File: src/WebUI/tsconfig.json
Comment (or2e) [inline src/WebUI/tsconfig.json:1]: just prettier (removed commas that are not valid for json)

File: src/WebUI/src/types/typed-router.d.ts
Comment (or2e) [inline src/WebUI/src/types/typed-router.d.ts:1]: It's not me, it's the auto generated

File: src/WebUI/package.json [hunk 3]
```diff
@@ -29,61 +28,57 @@
...
   "devDependencies": {
-    "@intlify/vite-plugin-vue-i18n": "^6.0.3",
-    "@pinia/testing": "^0.0.16",
+    "@intlify/unplugin-vue-i18n": "^0.10.0",
+    "@pinia/testing": "^0.1.1",
     "@tailwindcss/typography": "^0.5.9",
     "@types/body-scroll-lock": "^3.1.0",
     "@types/itemsjs": "^2.1.3",
     "@types/luxon": "^3.3.0",
-    "@types/node": "^18.16.3",
```
Comment (or2e) [inline src/WebUI/package.json:48]: deleted it for now, I don't think it's needed.

**Review by verdie-g — APPROVED** [review]

**Pull request merged**